starter-code/src/functions-and-arrays.js

```diff
@@ -1,30 +1,56 @@
 // Iteration #1: Find the maximum
+const maxOfTwoNumbers = (firstNum, secondNum) => (firstNum > secondNum ? firstNum : secondNum);
```

Comment:
interesting use of the reduce and of the ternary. You could have used Math.max(firstNum, secondNum) and that would have given you the response right away